simple/dgram.c

```diff
@@ -181,6 +181,13 @@ static int bind_ep_res(void)
 		return ret;
 	 }
 
+	ret = fi_recv(ep, buf, buffer_size, fi_mr_desc(mr), 0,
```

Comment:
Probably not, but I think we may want all tests to call this function, unless we have some other way of ensuring that all data has been transferred, such as using and waiting for shutdown.